### PR TITLE
BrenKt-SK964-WEB-Exchange-restore_email

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by BrenKt** [comment]
Only translations

File: pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.es-es.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+A continuación, acepte la restauración haciendo clic en `Obtener`{.action}.
+
+> [!primary]
+>
+> Es posible seleccionar diferentes tipos de elementos simultáneamente y restaurarlo. Cada tipo de elemento se situará en la categoría correspondiente.
+> 
+
+## Más información
+ 
+Interactúe con nuestra comunidad de usuarios en <https://community.ovh.com/>.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.es-es.md]:
```suggestion
Interactúe con nuestra comunidad de usuarios en <https://community.ovh.com/en/>.
```

File: pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.it-it.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Cliccando su `Recupera`{.action}.
+
+> [!primary]
+>
+> È possibile selezionare diversi tipi di elementi simultaneamente e effettuare il ripristino. Ciascun tipo di elemento sarà quindi classificato nella categoria che lo riguarda.
+> 
+
+## Per saperne di più
+ 
+Contatta la nostra Community di utenti all’indirizzo <https://community.ovh.com/>.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.it-it.md]:
```suggestion
Contatta la nostra Community di utenti all’indirizzo <https://community.ovh.com/en/>.
```

File: pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.pl-pl.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+Zatwierdź przywrócenie, klikając `Przywróć`{.action}.
+
+> [!primary]
+>
+> Możliwe jest jednoczesne wybieranie różnych rodzajów elementów i przywrócenie. Każdy typ elementu zostanie następnie umieszczony w odpowiedniej kategorii.
+> 
+
+## Sprawdź również
+ 
+Dołącz do społeczności naszych użytkowników na stronie<https://community.ovh.com/>.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.pl-pl.md]:
```suggestion
Dołącz do społeczności naszych użytkowników na stronie<https://community.ovh.com/en/>.
```

File: pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.pt-pt.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+De seguida, valide o seu restauro clicando em `Recuperar`{.action}.
+
+> [!primary]
+>
+> É possível selecionar diferentes tipos de elementos simultaneamente e efetuar o restauro. Cada tipo de elemento será então colocado na categoria que lhe diz respeito.
+> 
+
+## Saiba mais
+ 
+Fale com a nossa comunidade de utilizadores em <https://community.ovh.com/>.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/diagnostic_retention/guide.pt-pt.md]:
```suggestion
Fale com a nossa comunidade de utilizadores em <https://community.ovh.com/en/>.
```

**Pull request merged**